starter-code/routes/passportRouter.js

```diff
@@ -11,7 +16,65 @@ const ensureLogin = require("connect-ensure-login");
 
 
 passportRouter.get("/private-page", ensureLogin.ensureLoggedIn(), (req, res) => {
```

Comment:
You correctly check and make sure the user is loggedIn before rendering the requested page, but I case the user is not loggedIn nothing happens.

Therefore the user lands on a 404 page. It's better for the user experience to redirect the user to the login or sign-up page when this happens.